consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/QbftContext.java

```diff
@@ -18,38 +18,21 @@
 import org.hyperledger.besu.consensus.common.bft.BftBlockInterface;
 import org.hyperledger.besu.consensus.common.bft.BftContext;
 import org.hyperledger.besu.consensus.common.validator.ValidatorProvider;
-import org.hyperledger.besu.consensus.qbft.pki.PkiBlockCreationConfiguration;
-
-import java.util.Optional;
 
 /** The Qbft context. */
 public class QbftContext extends BftContext {
 
-  private final Optional<PkiBlockCreationConfiguration> pkiBlockCreationConfiguration;
-
   /**
    * Instantiates a new Qbft context.
    *
    * @param validatorProvider the validator provider
    * @param epochManager the epoch manager
    * @param blockInterface the block interface
-   * @param pkiBlockCreationConfiguration the pki block creation configuration
    */
   public QbftContext(
```

Comment:
nit: QBFT could be changed to use the BftContext instead, there was only a QBFT specific context because of PKI block creation. But this would be in a lot place so probably doesn't make sense in this PR

Reply:
ok yeah that could be a separate PR also